### ISSUE TITLE
ADE20K segmaps not correctly labeled

### ISSUE THREAD

**Issue by SidShenoy** [Vladkryvoruchko/PSPNet-Keras-tensorflow#64]
When I used the '-m pspnet50_ade20k' option, the RGB segmaps generated were not correctly labeled/colored. I found out a fix for this,

Line number 209 of 'pspnet.py' must be changed as follows,

`import ade20k_labels`  
`color_cm = utils.class_image_to_image(cm,ade20k_labels.labels)`

Similar changes must be made according to the other options `-m 'pspnet101_cityscapes' OR 'pspnet101_voc2012'` chosen by the user.
